test/helpers/index.js

```diff
@@ -8,6 +8,82 @@ const ERC721_CONTRACT_NAME = "./ERC721Patronage_v1.sol";
 const ERC20_CONTRACT_NAME = "./ERC20PatronageReceipt_v2.sol";
```

Comment:
All the changes in this helpers file are used a lot in the tests; you will get a lot of context going through those.